source/addComment.js

```diff
@@ -1,3 +1,26 @@
 $(document).ready(function(){
   // add your code here
+  $("button#new_comment_button").click(function() {
+    event.preventDefault();
+    $("button#new_comment_button").hide();
```

Comment:
With unique CSS IDs, you can drop the `button` and just use `#new_comment_button`.